bigtable-dataflow-parent/bigtable-hbase-beam/src/main/java/com/google/cloud/bigtable/beam/CloudBigtableIO.java

```diff
@@ -678,17 +678,17 @@ public final Double getFractionConsumed() {
     }
 
     @VisibleForTesting
-    protected void setSession(BigtableSession session) {
+    void setSession(BigtableSession session) {
```

Comment:
why are we changing the visibility of these methods?

Reply:
As I can see, these methods are introduced for the sake of testability, they are not supposed to be invoked not by tests,
so there is no reason for them to belong to the public API.

Reply:
Got it - since this is not required for this fix, let's revert these and maybe revisit the public surface in a different issue/PR just in case this would inadvertently break something :)

Reply:
Yes, I agree, fixing multiple issues in the same commit is not a good practice.
Restored, please take a look when you have time